docs/apis-tools/java-client/job-worker.md

```diff
@@ -125,10 +125,23 @@ public final JobWorker openWorker(final CamundaClient client, final JobHandler h
 There are currently no built-in tags, primarily because these are likely to be high cardinality, which can become an issue with some metric registries. If you want per-worker tags, create a different `JobWorkerMetrics` instance per worker.
 :::
 
-This implementation creates two metrics:
+This implementation creates four metrics:
 
-- `zeebe.client.worker.job.activated`: A counter tracking the count of jobs activated.
-- `zeebe.client.worker.job.handled`: A counter tracking the count of jobs handled.
+| Metric Name                         | Description                           | Status     | Replacement                         | Removal Version |
+| ----------------------------------- | ------------------------------------- | ---------- | ----------------------------------- | --------------- |
+| camunda.client.worker.job.activated | Counts the number of jobs activated   | New        | -                                   | -               |
+| camunda.client.worker.job.handled   | Counts the number of jobs handled     | New        | -                                   | -               |
+| zeebe.client.worker.job.activated   | Deprecated counter for jobs activated | Deprecated | camunda.client.worker.job.activated | 8.10            |
+| zeebe.client.worker.job.handled     | Deprecated counter for jobs handled   | Deprecated | camunda.client.worker.job.handled   | 8.10            |
+
+:::warning Deprecated Metrics
+The following metrics are deprecated and will be removed in version 8.10:
```

Comment:
Do we mean the 8.8 release or a particular alpha?

Reply:
we will have these changes in the next alphas (maybe not in the July one, because the code is already frozen), but customers can still use the old client as we currently planned to remove the code with 8.10 (maybe early alphas)